commit 4b650477840e3c602f63640dbe970fd86e0fb7ef
Author:     Yuhong Wen <yuhongw@nvidia.com>
AuthorDate: Tue Aug 30 12:16:36 2022 -0400
Commit:     GitHub <noreply@github.com>
CommitDate: Tue Aug 30 12:16:36 2022 -0400

    Fixed typos and return run status for simulator (#803)
    
    * Fixed issues raised by QA.
    
    * Removed the no need --set option.
    
    * Removed the no need import.
    
    * Fixed the unittest.
---
 nvflare/private/fed/client/client_executor.py | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/nvflare/private/fed/client/client_executor.py b/nvflare/private/fed/client/client_executor.py
index 0c987a0b..167845b2 100644
--- a/nvflare/private/fed/client/client_executor.py
+++ b/nvflare/private/fed/client/client_executor.py
@@ -24,7 +24,6 @@ from multiprocessing.connection import Client
 from nvflare.apis.fl_constant import AdminCommandNames, ReturnCode, RunProcessKey
 from nvflare.apis.resource_manager_spec import ResourceConsumerSpec, ResourceManagerSpec
 from nvflare.apis.shareable import Shareable, make_reply
-from nvflare.fuel.utils.pipe.file_pipe import FilePipe
 
 from .client_status import ClientStatus, get_status_message
 
@@ -37,11 +36,6 @@ class ClientExecutor(object):
             uid: client name
             startup: startup folder
         """
-        pipe_path = startup + "/comm"
-        if not os.path.exists(pipe_path):
-            os.makedirs(pipe_path)
-
-        self.pipe = FilePipe(root_path=pipe_path, name="training")
         self.logger = logging.getLogger(self.__class__.__name__)
 
     def start_train(
@@ -141,7 +135,7 @@ class ClientExecutor(object):
 
     def cleanup(self):
         """Cleanup."""
-        self.pipe.clear()
+        pass
 
 
 class ProcessExecutor(ClientExecutor):
